<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleShapeDialog</class>
 <widget class="QDialog" name="SimpleShapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleShapeDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="x_label">
      <property name="text">
       <string>x position</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="x_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="y_label">
      <property name="text">
       <string>y position</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="y_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="speed">
      <property name="text">
       <string>speed      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="v_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>x_lineEdit</tabstop>
  <tabstop>y_lineEdit</tabstop>
  <tabstop>v_lineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SimpleShapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SimpleShapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
